index.js

```diff
@@ -91,6 +100,28 @@ function _insertCredentials(username, userhash) {
 let cachedAgents;
 
 /**
+ * Generates and maps an internal username to the given username
+ *
+ * To protect against username collisions, this function takes
+ * a passed in username and adds a hash to it, generating an
+ * internal username.
+ *
+ * @param {string} username Username of the user.
+ *
+ * @return {string} Internal username
+ *
+ * @private
+ */
+function _generateInternalUsername(username) {
+    if (usernameMap[username]) {
+        throw new Error(`User ${username} already exists`);
```

Comment:
I removed the unnecessary comment, compared to https://github.com/sugarcrm/thorn/pull/122/files#diff-168726dbe96b3ce427e7fedce31bb0bcR344